src/Language/PureScript/AST/Binders.hs

```diff
@@ -14,78 +14,90 @@ import Language.PureScript.Types
 -- |
 -- Data type for binders
 --
-data Binder
+data Binder a b
```

Comment:
You may, but `type` is an illegal name, and `b` is for value annotations in general 😜 

Point taken though... I'll need to find something to help me do the refactoring. It took me ~5 hours to get this far even!